### DISCUSSION TITLE
What causes "attempt to yield across metamethod/C-call boundary"?

### DISCUSSION THREAD

**Discussion by omar2205** [williamboman/nvim-lsp-installer#665] [Q&A] [answered]
## nvm
It's happening with `:Telescope find_files` too, Idk what's the problem. So, delete this thread.

was using nvim-8 rolled back to nvim-7 and it worked.

<hr>

Every server I try to install I get "attempt to yield across metamethod/C-call boundary"
```
Pending servers (1)
◍ tsserver (failed)
attempt to yield across metamethod/C-call boundary
```

Tried with this simple config
```
-- init.lua
require("nvim-lsp-installer").setup {}
```

```
" p.vim (sourced in init.vim)

" --- LSP autocomplete                 
" LSP Support                          
Plug 'neovim/nvim-lspconfig'           
Plug 'williamboman/nvim-lsp-installer' 
                                       
" Autocompletion                       
Plug 'hrsh7th/nvim-cmp'                
Plug 'hrsh7th/cmp-buffer'              
Plug 'hrsh7th/cmp-path'                
Plug 'saadparwaiz1/cmp_luasnip'        
Plug 'hrsh7th/cmp-nvim-lsp'            
Plug 'hrsh7th/cmp-nvim-lua'            
                                       
" Snippets                             
Plug 'L3MON4D3/LuaSnip'                
Plug 'rafamadriz/friendly-snippets'    

```

**Comment by williamboman** [comment, marked as answer]
Hello! Hmm that shouldn't happen. There's an [issue template](https://github.com/williamboman/nvim-lsp-installer/issues/new?assignees=&labels=installation-issue&template=server_installation_form.yaml) for problems with installing servers if you wish to dig deeper into the issue

**Reply by omar2205** [reply]
It was because I somehow was using nvim 8.0, rolled back to 7 and it worked.
